src/handlers/scheduledEventHandler.ts

```diff
@@ -94,3 +95,50 @@ export const addMissedUpdatesRole = async (env: env) => {
 		console.error('Error while adding missed updates roles');
 	}
 };
+
+export const syncUsersStatus = async (env: env): Promise<any | null> => {
+	await apiCaller(env, 'users/status/update', 'PATCH');
+
+	try {
+		const idleUsersData = (await apiCaller(env, 'users/status?aggregate=true', 'GET')) as UserStatusResponse | undefined;
+
+		if (!idleUsersData?.data?.users || idleUsersData.data.users.length === 0) {
+			console.error('Error: Users data is not in the expected format or no users found');
+			return null;
+		}
+
+		const response = await apiCaller(env, 'users/status/batch', 'PATCH', {
+			method: 'PATCH',
```

Comment:
I can see PATCH being specified 2 times. Is this expected?

Reply:
Thanks, fixed